charts/linkerd2-cni/Chart.yaml

```diff
@@ -9,4 +9,4 @@ description: |
 kubeVersion: ">=1.21.0-0"
 icon: https://linkerd.io/images/logo-only-200h.png
 name: "linkerd2-cni"
-version: 30.12.0-edge
+version: 30.12.1-edge
```

Comment:
There were no changes to the linkerd2-cni chart, so no need to bump this one.

Reply:
(sorry, RELEASE.md is a bit outdated about this)

Reply:
whoops, i'll remove it!